web-bluetooth/scan.js

```diff
@@ -42,3 +54,16 @@ async function onButtonClick() {
     log('Argh! ' + error);
   }
 }
+
+/* Utils */
+
+const valueDataLogger = (key, valueDataView, whatData) => {
+  const hexString = [...new Uint8Array(valueDataView.buffer)].map(b => {
+    return b.toString(16).padStart(2, '0');
+  }).join('');
```

Comment:
I recommend `join(' ')` so that the bytes can be visually parsed.

Reply:
Done.